src/CRUDlex/EntityDefinition.php

```diff
@@ -557,6 +563,26 @@ public function setInitialSortAscending($initialSortAscending) {
     public function isInitialSortAscending() {
         return $this->initialSortAscending;
     }
+    
+    /**
+     * Gets the navigation bar group where the entity belongs
+     *
+     * @return string
+     * the navigation bar group where the entity belongs
+     */
+    public function getNavBarGroup() {
+        return $this->navBarGroup;
+    }
+
+    /**
+     * Sets the navigation bar group where the entity belongs
+     *
+     * @param string $navBarGroup
+     * the navigation bar group where the entity belongs
+     */
+    public function setNavBarGroup($navBarGroup) {
```

Comment:
This method should get an explicit test: EntityDefinitionTest::testSetNavBarGroup